l10n_eu_nace/models/res_partner.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+
+    _inherit = 'res.partner'
+
+    nace_id = fields.Many2one(comodel_name="res.partner.nace", string="NACE")
```

Comment:
Always index these: https://github.com/odoo/odoo/pull/39332

```suggestion
    nace_id = fields.Many2one(comodel_name="res.partner.nace", string="NACE", index=True)
```